docs-src/docs/reference/setup-cli.md

```diff
@@ -68,6 +68,7 @@ $ l0-setup apply [options] <instance_name>
 
 ### Options
 * `--quick` - Skips verification checks that normally run after `terraform apply` has completed
+* `--push` - Skips uploading local tfstate file to s3. (default: true)
```

Comment:
I incorrectly assumed we weren't exposing that detail to the user but checking the documentation [push](http://layer0.ims.io/reference/setup-cli/#push), it looks like we do. Will update to match description already in documentation.

> The push command is used to back up your Layer0 configuration files to an S3 bucket.